packages/eslint-plugin/package.json

```diff
@@ -13,6 +13,14 @@
         "test": "jest",
         "test:watch": "jest --watch"
     },
+    "dependencies": {
+        "@typescript-eslint/parser": "^5.0.0",
+        "eslint-config-prettier": "^8.6.0",
+        "eslint-plugin-prettier": "^4.2.1",
+        "eslint-plugin-simple-import-sort": "^9.0.0",
+        "eslint-plugin-unused-imports": "^2.0.0",
+        "@typescript-eslint/eslint-plugin": "^5.48.2"
+    },
```

Comment:
Why does the plugin need the dependencies?

Reply:
i think, they should be dev dependencies.